.github/workflows/sentry_sidekiq_test.yml

```diff
@@ -60,12 +66,7 @@ jobs:
           redis-version: ${{ matrix.sidekiq_version == '7.0' && 6 || 5 }}
 
       - name: Run specs with Sidekiq ${{ matrix.sidekiq_version }}
-        env:
-          SIDEKIQ_VERSION: ${{ matrix.sidekiq_version }}
-          RUBYOPT: ${{ matrix.options.rubyopt }}
-        run: |
-          bundle install --jobs 4 --retry 3
-          make test
+        run: make test
```

Comment:
I think we can move `make test`'s definitions here and remove it as other Makefiles don't seem to have it anymore either.